ui/shared/components/graph/VariantTagTypeBar.jsx

```diff
@@ -5,22 +5,27 @@ import HorizontalStackedBar from '../graph/HorizontalStackedBar'
 import { EXCLUDED_TAG_NAME, REVIEW_TAG_NAME, NOTE_TAG_NAME } from '../../utils/constants'
 
 
-export const getVariantTagTypeCount = (vtt, familyGuids) => (
-  familyGuids ? familyGuids.reduce((count, familyGuid) => count + (vtt.numTagsPerFamily[familyGuid] || 0), 0) : vtt.numTags
-)
+export const getVariantTagTypeCount = (vtt, familyGuids, variantGuid, variantsToDisplay) => {
```

Comment:
You have to be very careful when you change the function signature for functions that are exported. This could be used elsewhere and break. Rather than adding new arguments here, I think you should change how `familyGuid` is chosen in the first place. So instead of passing `variantGuid` and `variantsToDisplay` into the graph component, just change the props thats passed in from 
```
familyGuid={familyGuid}
variantGuid={variantGuid}
variantsToDisplay={this.props.variantsToDisplay}
```
to
```
familyGuid={variantGuid ?  (this.props.variantsToDisplay[0] || {}).familyGuids || [])[0] :  familyGuid}
```